src/ucp/wireup/select.c

```diff
@@ -56,23 +56,26 @@ typedef struct {
 } ucp_wireup_lane_desc_t;
 
 
-typedef struct {
-    ucp_wireup_criteria_t criteria;
-    uint64_t              local_dev_bitmap;
-    uint64_t              remote_dev_bitmap;
-    ucp_md_map_t          md_map;
-    unsigned              max_lanes;
-} ucp_wireup_select_bw_info_t;
-
-
 typedef struct {
     unsigned local[UCP_MAX_RESOURCES];
     unsigned remote[UCP_MAX_RESOURCES];
     unsigned local_skip[UCP_MAX_RESOURCES];
     unsigned remote_skip[UCP_MAX_RESOURCES];
+    ucp_lane_index_t excl_lane;
+    ucp_rsc_index_t skip_dev_index;
 } ucp_wireup_dev_usage_count;
 
 
+typedef struct {
+    ucp_wireup_criteria_t criteria;
+    uint64_t                   local_dev_bitmap;
+    uint64_t                   remote_dev_bitmap;
+    ucp_md_map_t               md_map;
+    unsigned                   max_lanes;
+    ucp_wireup_dev_usage_count *dev_count;
```

Comment:
why needed? it's already in criteria.arg
then will also not need to move this sruct definition